GeometrickFX/Shapes/AbstractDomain/IShape.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shapes.AbstractDomain
+{
+    public interface IShape
+    {
+        float Area();
+        float[] Localize();
```

Comment:
over-complication; brings only complexity without fulfilling the requirements

Reply:
removed